charts/spire/charts/spire-server/values.yaml

```
@@ -182,6 +182,13 @@ controllerManager:
 
   validatingWebhookConfiguration:
     failurePolicy: Fail
+    upgradeHook:
+      image:
+        registry: "bitnami"
+        repository: kubectl
+        pullPolicy: IfNotPresent
+        # Overrides the image tag whose default is the chart appVersion.
+        version: "latest"
```

https://www.chainguard.dev/unchained/chainguard-image-now-available-for-kubectl
https://github.com/chainguard-images/images/tree/main/images/kubectl

```suggestion
        registry: cgr.dev
        repository: chainguard/kubectl
        pullPolicy: IfNotPresent
        version: latest
```